package.json

```diff
@@ -35,15 +35,15 @@
   "main": "./lib/nodelint.js",
   "dependencies": {
     "exit": "~0.1.2",
-    "glob": "^7.0.3",
+    "glob": "~7.1.2",
     "nopt": "~3.0.1",
-    "readable-stream": "~2.1.2"
+    "readable-stream": "~2.1.5"
   },
   "devDependencies": {
     "fs.extra": "^1.3.2",
-    "istanbul": "^0.4.3",
-    "marked-man": "~0.1.5",
-    "mocha": "^3.0.0"
+    "istanbul": "~0.4.5",
+    "marked-man": "~0.1.6",
+    "mocha": "~3.4.2"
```

Comment:
This wasn't an intentional change -- this is what `npm update` gave me - I think the default used to be to preserve ^ or ~, but now they're switching to `~` -- let me see if I can find when & where they changed this.